docs/renewCertificates.md

```diff
@@ -2,7 +2,45 @@
 
 The following instructions will guide you on how to renew  the existing ssl certificates for the ngsa-asb setup.
 
-## Create Certificate bundle
+## Provision certificates
+
+### Check expiration date
+
+You can check the expiration date of the deployed certificate from the website. This can also be used in automation to check when to renew a certificate, in order to avoid downtime or hitting rate limits.
+
+```bash
+
+# get the expiration date of the certificate directly from the site
+EXPIRATION_DATE=$(echo | \
+  openssl s_client -connect grafana-eastus-dev.austinrdc.dev:443 -servername austinrdc.dev 2> /dev/null | \
+  openssl x509 -noout -enddate | \
+  cut -d'=' -f2)
+
```

Comment:
Here I would add a conditional to check if the EXPIRATION_DATE is in the right format or if was not retrieved at all. Exit if couldn't be retrieved

 Something like this (make sure to test in case  =)  )

if ! [[ "$EXPIRATION_DATE" =~ ^[A-Z][a-z]{2}\ [0-9]{1,2}\ [0-9]{2}:[0-9]{2}:[0-9]{2}\ [0-9]{4}\ GMT$ ]]; then
    echo "Invalid SSL certificate expiration date: $EXPIRATION_DATE"
    exit 1
fi